### ISSUE TITLE
Minor issue with queued builds and missing notification toasts

### ISSUE THREAD

**Issue by rsuk** [mattwendels/tfs-build-notifications#6]
Thank you so much for such a useful tool. I'm sure it would be useful to a huge number of people if it only came up higher in the google searches.

I'm having a couple of minor issues. Firstly, I'm not seeing any notification "toasts" - the dashboard shows the correct info, but the notifications themselves aren't popping up when builds complete. I'm running Windows 10, and all the builds are hosted on VSTS / Azure Devops. Is this a know issue? If not, I will try running up the source and see if I can figure out what is going on.

Secondly, the dashboard is not showing the correct status for queued builds - they are showing in gray and when you hover over them the status shows "Unknown". Again, if this isn't something you already know about I can run the source and see if I can spot the issue.

**Comment by mattwendels** [comment]
Hi @rsuk - thanks for your comments.

Re the toast notifications, the implementation isn't well tested on all editions/updates of Win 10 - in fact, since my PC upgraded recently the toast notifications look a bit different (not quite right) so probs needs some attention.

In the meantime, in the app.config file can you enable Tooltip notifications instead? See if that works. At least then you'll know that the notifications system is actually working.

Re queued build status... how are you connecting to TFS? On-prem or via VSTS online etc. ? Which version of TFS? I developed this against TFS 2017, it's v. possible the APIs have changed in later versions.

Thanks!

**Comment by rsuk** [comment]
I'm using VSTS online (Azure Devops). I will do a bit of digging in debug and see what I can uncover. Will try your suggestion on Tooltip notifications too and let you know what I find.

**Comment by rsuk** [comment]
Tooltip notifications works just fine. It looks like the Windows 10 fall creators update closed a loophole that previously allowed Toast Notifications to work without a shortcut with a matching app id: https://stackoverflow.com/questions/46814858/toast-notification-not-working-on-windows-fall-creators-update

Are you looking for contributors to help with this project? I could investigate creating a basic wix setup and that might also enable a fix for the toast issue.

In the meantime I have submitted a pull request [7](https://github.com/mattwendels/tfs-build-notifications/pull/7) to add an InProgress status for incomplete builds. The PR also contains a proposed fix for a problem I encountered with missing build completion notifications caused by slight date/time differences between the build server and the local machine.
